src/library/scala/collection/Iterator.scala

```diff
@@ -212,44 +211,39 @@ trait Iterator[+A] extends IterableOnce[A] with IterableOnceOps[A, Iterator, Ite
       buf
     }
 
-    private def padding(x: Int) = immutable.ArraySeq.untagged.fill(x)(pad.get())
-    private def gap = (step - size) max 0
+    private def gap =
+      (step - size) max 0
```

Comment:
why is this on a new line?

Reply:
No good reason, just preference - I can put it back on one line

Reply:
because it adds a "gap".